### PR TITLE
Ports several Paradise sound emotes

### PR DESCRIPTION
**Pull request by Arokha**
In total this adds or updates with sound: sneeze, cough, rsneeze, rcough, rscream, snap, slap.

Split robo-cough and robo-sneeze and robo-scream into their own *rcough, rsneeze, rscream because we don't really have it as a species, and their logic of only allowing the species to do it wouldn't work. It checks if you're synthetic like the existing synthonly emotes.

They have these as species vars so I did that. If someone wants to get cat sneeze noises for Tajarans or something, you know. Feel free.

The scream sound is commented out for now... the sound just doesn't... sound right for HRP servers? Maybe someone can find a better scream sound. These seem way too long.

No superfart. "Sorry" ~~notsorry~~

### CODE REVIEW

**Comment by nachomeep** [comment]
>no *fart

get out

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -565,18 +602,58 @@
...
 					message = "sadly can't find anybody to give daps to, and daps [get_visible_gender() == MALE ? "himself" : get_visible_gender() == FEMALE ? "herself" : "themselves"]. Shameful."
 
-		if ("scream")
-			if (miming)
+		if("slap", "slaps")
+			m_type = 1
+			if(!restrained())
+				var/M = null
+				if(param)
+					for(var/mob/A in view(1, null))
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: emotes: allowing me to spam view() as fast as i can hit the keyboard macro

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -597,18 +674,14 @@
 			return
 
 		if ("help")
-			src << {"blink, blink_r, blush, bow-(none)/mob, burp, choke, chuckle, clap, collapse, cough, cry, custom, deathgasp, drool, eyebrow, fastsway/qwag,
-frown, gasp, giggle, glare-(none)/mob, grin, groan, grumble, handshake, hug-(none)/mob, laugh, look-(none)/mob, moan, mumble, nod, pale, point-atom,
-raise, salute, shake, shiver, shrug, sigh, signal-#1-10, smile, sneeze, sniff, snore, stare-(none)/mob, stopsway/swag, sway/wag, swish, tremble, twitch,
-twitch_v, vomit, whimper, wink, yawn"}
+			src << "blink, blink_r, blush, bow-(none)/mob, burp, choke, chuckle, clap, collapse, cough, cry, custom, deathgasp, drool, eyebrow, fastsway/qwag, \
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: this is shit

fix it

datum emotes

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 3]
```diff
@@ -50,6 +50,15 @@
 	var/num_alternate_languages = 0          // How many secondary languages are available to select at character creation
 	var/name_language = LANGUAGE_GALCOM    // The language to use when determining names for this species, or null to use the first name/last name generator
 
+	//Soundy emotey things.
+	var/scream_verb = "screams"
+	var/male_scream_sound = 'sound/goonstation/voice/male_scream.ogg'
+	var/female_scream_sound = 'sound/goonstation/voice/female_scream.ogg'
+	var/male_cough_sounds = list('sound/effects/mob_effects/m_cougha.ogg','sound/effects/mob_effects/m_coughb.ogg', 'sound/effects/mob_effects/m_coughc.ogg')
+	var/female_cough_sounds = list('sound/effects/mob_effects/f_cougha.ogg','sound/effects/mob_effects/f_coughb.ogg')
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/species.dm]: make 'em static

Comment (Arokha) [inline code/modules/mob/living/carbon/human/species/species.dm]: I'm unsure what you mean by "make 'em static". Do you mean only have one sound for each?

File: code/modules/mob/living/carbon/human/emote.dm (hunk 2 above)
Comment (Arokha) [inline code/modules/mob/living/carbon/human/emote.dm]: Converting the entire emote system to datums is outside the scope.

File: code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (Arokha) [inline code/modules/mob/living/carbon/human/emote.dm]: I can fix that. There should really be a cooldown for emotes, though, as on Paradise.

Comment (Arokha) [inline code/modules/mob/living/carbon/human/emote.dm]: Ok. I just looked, and there are like 10+ other emotes that have this. This is not new.

**Comment by Arokha** [comment]
I'm a little confused by all of your comments, @Iamgoofball, considering none of those 3 things you commented on are unique or changed by this PR.

File: code/modules/mob/living/carbon/human/species/species.dm (hunk 3 above)
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/species.dm]: var/static/list/fuck = meme()

File: code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: doesn't make it any less shit,

add a cooldown

**Comment by Anewbe** [comment]
I'm not sure how this'll play with our license, which is delaying the process a bit.

**Comment by Arokha** [comment]
That's fair. In looking up stuff online, it seems like GPL and CC BY-SA have an interesting relationship.

**Comment by Arokha** [comment]
I assume you mean the sounds, anyway? I can find some other license sounds, probably. Maybe. What license would be compatible? I doubt I'll find "AGPL" licensed sounds anywhere.

**Comment by Anewbe** [comment]
The sounds are the issue, if you can find free use sounds those shouldn't be a problem.

**Comment by Arokha** [comment]
Unfortunately in the US there is no actual working legal way to release anything into the public domain, so it'll be under some license, and I'd need to know what's compatible! It seems like CC BY-SA 4.0 (the goon sounds use 3.0) is apparently one-way compatible INTO GPLv3. So that would probably work. CC0 would also work probably.

Does Bay or any other (A)GPLv3 server have good sounds for this stuff?

**Comment by Iamgoofball** [comment]
>no way to release into public domain

i think all the furry porn got to your logical thinking center

try again

**Comment by Arokha** [comment]
I removed the creative commons license sounds. The rest should just be AGPLv3 due to being ported from other AGPLv3 codebases.

**Comment by Anewbe** [comment]
<3 Will re-review soonish.

**Pull request merged**